### PR TITLE
Hotfixes, text changes and layout upgrade

### PR DESCRIPTION
**Pull request by ioaNikas**
Text changes, bug fixes and some css

### CODE REVIEW

File: libs/organization/src/lib/organization/forms/organization-form/organization-form.component.html [hunk 1]
```diff
@@ -10,9 +10,10 @@
           <mat-form-field appearance="outline">
             <!-- Here is missing an input for the public name of an organization,
             we always have to decide how we will handle that -->
-            <mat-label>Company Name</mat-label>
+            <mat-label>Company Name *</mat-label>
```
Comment (GrandSchtroumpf) [inline libs/organization/src/lib/organization/forms/organization-form/organization-form.component.html:13]: Add "require in the input and it'll put the "*" automatically.

File: libs/user/src/lib/auth/components/signin-form/signin-form.component.scss [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 section {
   height: 100%;
-  margin: 100px 100px 100px 24px;
+  margin: 100px 100px 100px 100px;
```
Comment (GrandSchtroumpf) [inline libs/user/src/lib/auth/components/signin-form/signin-form.component.scss:5]: just put `margin: 100px`

File: libs/user/src/lib/auth/components/signup-form/signup-form.component.scss [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 section {
   height: 100%;
-  margin: 100px 24px 100px 100px;
+  margin: 100px 100px 100px 100px;
```
Comment (GrandSchtroumpf) [inline libs/user/src/lib/auth/components/signup-form/signup-form.component.scss:5]: same

File: libs/movie/src/lib/movie/form/filters/budget/budget.component.html [hunk 4]
```diff
@@ -1,8 +1,9 @@
 <mat-slider
```
[the remark this answers is not in the document]
Comment (ioaNikas) [reply libs/movie/src/lib/movie/form/filters/budget/budget.component.html:1]: Moved back to earlier design, added a customization file in style folder to change thumb and thumb text sizes.

File: libs/movie/src/lib/movie/form/filters/budget/budget.component.ts [hunk 5]
```diff
@@ -11,14 +12,17 @@ export class BudgetFilterComponent {
 
   @Input() form: FormControl; // FormControl of number
 
-  formatLabel(value: number) {
+  public currentValue = '0';
+
+  updateCurrentValue(input: MatSliderChange) {
+    const value = input.value
     if (value >= 1000 && value < 1000000) {
```
[the remark this answers is not in the document]
Comment (ioaNikas) [reply libs/movie/src/lib/movie/form/filters/budget/budget.component.ts:19]: Rolled back to earlier version of it, won't use this

File: libs/movie/src/lib/movie/layout/list/title-list.component.html [hunk 6]
```diff
@@ -84,10 +87,14 @@ <h6>There are {{ titles.length }} {{titleType}}s available!</h6>
...
           </ng-template>
         </div>
-      </ng-container>
     </ng-container>
     <ng-template #loading>
       <mat-progress-spinner mode="indeterminate" color="primary"></mat-progress-spinner>
     </ng-template>
+    <ng-template #empty>
+      <div fxLayout="row" fxLayoutAlign="end start">
+        <h6 *ngIf="!titles.length">There is no corresponding title.</h6>
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/layout/list/title-list.component.html:96]: You don't need to verify this here

File: libs/movie/src/lib/movie/layout/list/title-list.component.ts [hunk 7]
```diff
@@ -27,12 +29,13 @@ export class TitleListComponent implements AfterContentInit {
...
   @ContentChild(TitleListItemDirective, { read: TemplateRef }) titleListItemTemplate: TitleListItemDirective;
 
-  @Input() titles$: Observable<any>;
+  @Input() titles: Movie[];
 
   @Input() titleType = 'title'; // only for display purpose
 
   public listView = false;
   public canToggle = false;
+  public isLoaded = of(false);
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/layout/list/title-list.component.ts:38]: Do you need that ?

**Pull request closed without merging**